nixos/modules/services/web-apps/freshrss.nix

```diff
@@ -116,12 +116,18 @@ in
         with default values.
       '';
     };
-  };
 
+    user = mkOption {
+      type = types.str;
+      default = "freshrss";
+      description = lib.mdDoc "User under which Freshrss runs.";
+    };
+  };
 
   config =
     let
-      systemd-hardening = {
+      defaultServiceConfig = {
+        ReadWritePaths = "$(cfg.dataDir)";
```

Comment:
```suggestion
        ReadWritePaths = "${cfg.dataDir}";
```


As is:

```
Jan 06 21:54:29 futaba systemd[1]: /etc/systemd/system/freshrss-updater.service:33: ReadWritePaths= path is not absolute, ignoring: $(cfg.dataDir)
Jan 06 21:54:29 futaba systemd[1]: /etc/systemd/system/freshrss-config.service:32: ReadWritePaths= path is not absolute, ignoring: $(cfg.dataDir)

$ grep ReadWritePaths /etc/systemd/system/freshrss-updater.service
ReadWritePaths=$(cfg.dataDir)
```

Reply:
Also more of a side note, but even with the red log text above things seem to be working (tested with sqlite). You may not need this bit at all.

Reply:
thanks. this is needed so we still have `ProtectSystem` set to strict.